projects/demo/src/modules/experimental/pin/pin.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {TuiDocExample, TuiRawLoaderContent} from '@taiga-ui/addon-doc';
+
+@Component({
+    selector: 'example-pin',
+    templateUrl: './pin.template.html',
+    changeDetection,
+})
+export class ExampleTuiPinComponent {
+    readonly exampleModule: TuiRawLoaderContent = import(
+        './examples/import/import-module.md?raw'
+    );
+
+    readonly exampleHtml: TuiRawLoaderContent = import(
+        './examples/import/insert-template.md?raw'
+    );
+
+    readonly example1: TuiDocExample = {
+        TypeScript: import('./examples/1/index.ts?raw'),
+        HTML: import('./examples/1/index.html?raw'),
```

Comment:
I think we could move all `HTML` tabs on the first place for all examples on this documentation page.

They look like the most important part.